chart/ci/externaldb-values.yaml

```diff
@@ -0,0 +1,38 @@
+# This is the default values for the chart to run in CI system and it can be used as
```

Comment:
small nit: let's maybe modify this comment to something like
```
This file is based on the ci/default-values.yml. The values has been modified to connect to an externally provisioned TimescaleDB instance.
```

Reply:
great suggestion!